script/landlongnode.ks

```diff
@@ -35,3 +48,13 @@ print "T+" + round(missiontime) + " Descent orbit burn: " + round(vom) + ", dv:"
 set nd to node(time:seconds + etam, 0, 0, deltav).
 add nd.
 
+// adjust for the body we are orbiting to rotate under us whilst we cruise to the descending node.
+set shiptonodedegrees to 360 * etam / body:rotationperiod.
+set adjustforshiptonodeseconds to ops * shiptonodedegrees / 360.

```

Comment:
should now read
set adjustforshiptonodeseconds to sin(90 - ABS(ORBIT:INCLINATION)) * ops * shiptonodedegrees / 360.